### DISCUSSION TITLE
Option to collapse the classes only when classes length is greater than X characters

### DISCUSSION THREAD

**Discussion by pzgureanu** [moalamri/vscode-inline-fold#88] [Ideas]
Thanks for your work. I think is a good idea to add an option to collapse by number of characters, because when an HTML item has 1-2 small classes, the collapsing is not needed.

How i think that this can work:
To get all classes and concatenate, after that if the length is greater than X  characters, to collapse.

**Comment by moalamri** [comment]
Hey @pzgureanu 
This is possible using regex, there's an example in the repo's main page

**Reply by pzgureanu** [reply]
Thanks, will try
